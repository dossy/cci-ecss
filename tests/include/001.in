#foo { color: red; }
@include "x/001.inc";
#bar { color: blue; }
