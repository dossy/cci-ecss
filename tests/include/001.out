#bar{color:blue;}
#foo{color:yellow;}
